vmc/resource_vmc_sddc.go

```diff
@@ -18,6 +19,14 @@ import (
 	"github.com/vmware/vsphere-automation-sdk-go/services/vmc/orgs/sddcs"
 )
 
+var storageCapacityMap = map[string]int{
```

Comment:
this is a map of string to int and not int64

Reply:
Agreed. I will make the change

Reply:
Done